### DISCUSSION TITLE
How do you enable autodiscovery of zigbee2mqtt groups in home assistant?

### DISCUSSION THREAD

**Discussion by pearj** [Koenkk/zigbee2mqtt#18003] [General]
I created a zigbee2mqtt group, and[ according to this article](https://home-assistant-guide.com/guide/zigbee-groups-from-zigbee2mqtt-now-auto-discovered-in-home-assistant/#how-to-create-zigbee-groups-using-zigbee2mqtt), that group should appear as a device in home assistant.

However after I create the group, I don't see any new mqtt devices in home assistant. Am I doing something wrong?

I'm running home assistant: 2023.6.1 and zigbee2mqtt [1.18.1](https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.18.1).

**Comment by Koenkk** [comment]
You are using an old z2m version.

If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Reply by pearj** [reply]
Oh right thanks so much, I've upgraded and I can see the group in devices in HA now, yay.
